src/components/Login/Login.js

```diff
@@ -113,7 +113,7 @@ const Login = ({ location, dependencies: { dopplerLegacyClient, sessionManager,
   const onSubmit = async (values, { setSubmitting, setErrors }) => {
     try {
       const result = await dopplerLegacyClient.login({
-        username: values[fieldNames.user],
+        username: values[fieldNames.user].trim(),
```

Comment:
@andresmoschini I summon you, any better idea than doing this? back end is taking litteraly what we send, if we send spaces it takes them as part of the email. In old login the trimming was done in the view.

Reply:
Sorry, I did not saw this one. 

I think that it is fine in this way, it cannot be _null_, right?